src/main/java/com/bakdata/kafka/KafkaConnectorSourceResetter.java

```diff
@@ -124,6 +124,9 @@ private static Converter createConverter(final Map<String, Object> kafkaConfig)
 
     @Override
     public void run() {
+        if(this.offsetTopic.isBlank()) {
```

Comment:
I don't think this can happen because of picocli? also there is a missing psace after if

Reply:
As far as I know, picocli doesn't do blank validations.